### PR TITLE
Spike: Eq 716 household relationship spike

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+[![Build Status](https://travis-ci.org/ONSdigital/eq-survey-runner.svg?branch=master)](https://travis-ci.org/ONSdigital/eq-survey-runner) [![Code Issues](https://www.quantifiedcode.com/api/v1/project/1709e9d582cc479a86568a043117d4d0/badge.svg)](https://www.quantifiedcode.com/app/project/1709e9d582cc479a86568a043117d4d0) [![codecov](https://codecov.io/gh/ONSdigital/eq-survey-runner/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-survey-runner) [![Dependency Status](https://gemnasium.com/badges/github.com/ONSdigital/eq-survey-runner.svg)](https://gemnasium.com/github.com/ONSdigital/eq-survey-runner) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/82e63fc5bc5c43e8ba1ba6d13bfb4243)](https://www.codacy.com/app/ONSDigital/eq-survey-runner)
```
Comment (collisdigital) [inline README.md:2]: I really like this change :+1:

Comment (benjefferies) [reply README.md:2]: I said I'd make the change. Whether it will go in from this branch is another story as it's a spike!

Comment (jonnyshaw89) [reply README.md:2]: I can't even see what the change is.

**Comment by ajmaddaford** [comment]
Closing this, spike has proven approach is viable, the story will be completed in a new branch.

**Pull request closed without merging**